recipe/meta.yaml

```diff
@@ -22,36 +22,268 @@ requirements:
     - hatchling >=1.12.2
   run:
     - python
+      # Upstream not require max pin, but geenlet quite unstable dependency and would be good have a max pin
     - greenlet >=1.0
     - dnspython >=1.15.0
-    # https://github.com/eventlet/eventlet/blob/0.39.0/eventlet/green/OpenSSL/crypto.py#L1
-    - pyopenssl
+
+# Mass failing testst due to greenlet FdListener('read', -1, …)
```

Comment:
Can you just skip running tests for windows and osx? This would simplify the recipe.
As far as I can see, upstream doesn't run tests for windows https://github.com/eventlet/eventlet/blob/master/.github/workflows/test.yaml, and osx support is subpar: https://github.com/eventlet/eventlet/issues/1068